spk/tvheadend/src/service-setup.sh

```diff
@@ -19,108 +19,94 @@ GROUP=sc-media
 service_postinst ()
 {
     # Encrypt password
-    wizard_password=`echo -n "TVHeadend-Hide-${wizard_password:=admin}" | openssl enc -a`
+    wizard_password=$(echo -n "TVHeadend-Hide-${wizard_password:=admin}" | openssl enc -a)
 
     # Edit the password configuration according to the wizard
-    sed -i -e "s/@password@/${wizard_password}/g" ${SYNOPKG_PKGDEST}/var/passwd/a927e30a755504f9784f23a4efac5109
+    sed -i -e "s/@password@/${wizard_password}/g" ${SYNOPKG_PKGVAR}/passwd/a927e30a755504f9784f23a4efac5109
```

Comment:
I just saw a post in my email (that I can no longer find) about:
`${SYNOPKG_PKGDEST}/var` vs `${SYNOPKG_PKGVAR}`
Where, depending of DSM version it points to `package/target/var` or just `package/var`.
Now I wonder, are we good with the changes in here?

Reply:
Yes, It's backwards compatible in the `service-setup` scrips only

Reply:
The setting isn't applied. It's another symptom of the change here: https://github.com/SynoCommunity/spksrc/commit/0eaccc5f0a45bfa91b505f7ed9750c987128c235#r49449742